src/LaboratoryResults-3-0/Test/Receive/2-1a-receive-panelNHG.xml

```diff
@@ -0,0 +1,184 @@
+            range="low"
+            rangevalue = "90"
+            unit= "mL/min/{1.73_m2}"
+            system="http://unitsofmeasure.org"
+            code="mL/min/{1.73_m2}"
+        />
+        <nts:include value="assert.observation.interpretation" direction="response" resource="Observation"
+            interpretation="Iets bovengemiddeld"
+        />
+        
+        <!-- Asserts for checking the second Observation to be compliant with the wiki page on
+            https://informatiestandaarden.nictiz.nl/wiki/Lab:1.0.0_Testgegevens_SturenResultaten-->     
+        <nts:include value="assert.identifier" direction="response" resource="Observation" system="urn:oid:2.16.840.1.113883.2.4.3.11.999.25.4298.2" identifier="22222"/>  
+        <nts:include value="assert.observation.valueq" direction="response" resource="Observation"
+            quantity = "135"
+            unit = "micromol/l"
+            system = "http://unitsofmeasure.org"
+            code="{micromol/l}"
+        />
+        <nts:include value="assert.observation.referencerange" direction="response" resource="Observation"
+            range="high"
+            rangevalue ="110"
+            unit="micromol/l"
+            system="http://unitsofmeasure.org"
+            code="{micromol/l}"
+        />
+        <nts:include value="assert.observation.referencerange" direction="response" resource="Observation"
+            range="low"
+            rangevalue = "60"
+            unit="micromol/l"
+            system="http://unitsofmeasure.org"
+            code="{micromol/l}"
+        />
+        <nts:include value="assert.observation.method" direction="response" resource="Observation"
+            system="http://snomed.info/sct"
+            code="258026004"
+            display="enzyme method"
+        />
+        <nts:include value="assert.observation.interpretation" direction="response" resource="Observation"
+            interpretation="Creatinine iets hoger dan normaal."
+        />
+        <nts:include value="assert.observation.interpretationflag" direction="response" resource="Observation"
+            system="http://snomed.info/sct"
+            code="281302008"
+            display="boven referentiebereik"
+        />
+        
+        <!--Asserts for checking the Specimen resource(s) to be compliant with the wiki page on
+            https://informatiestandaarden.nictiz.nl/wiki/Lab:1.0.0_Testgegevens_SturenResultaten-->  
+        <nts:include value="assert.identifier" direction="response" resource="Specimen" system="urn:oid:2.16.840.1.113883.2.4.3.11.999.25.4267.2" identifier="1111"/>
+        <action>
+            <assert>
+                <description
+                    value="Confirm the observations have been sent in a panel"/>
+                <direction value="response"/>
+                <expression value="Bundle.entry.resource.as(Specimen).note.where(text='Serum afgenomen bij patiënt.').exists()"/>
```

Comment:
Does it make sense to create a component for this assert, or is that somewhat overkill?